ethereum/docs/src/design/security.md

```diff
@@ -1 +1,23 @@
-# Security considerations
\ No newline at end of file
+# Security considerations
+
+## Codebase
+
+Please note that the current code for the programs is not audited and should not be used in production.
+The code is provided as a reference implementation and should be used at the users risks.
+
+## Sphinx
+
+The [Sphinx](https://github.com/argumentcomputer/sphinx) prover is a fork of [SP1](https://github.com/succinctlabs/sp1)
+and as such inherits a lot from its security design. The current release of Sphinx (`dev`) has backported all the
+upstream security fixes as of SP1 `v1.0.8-testnet`. We will continue to update Sphinx with backports of upstream
+security fixes and subsequent updates to both Sphinx and the Light Client, making them available as hotfixes.
+
+In terms of Sphinx-specific changes that require special attention, here is a non-exhaustive list of Sphinx
```

Comment:
Are there any plans to audit this subset (deviation from SP1)?

Reply:
At this stage, we are rebasing our work on a recent version of SP1 so as to minimize the delta, which seems the sensible prelude to such work.